automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala

```diff
@@ -52,6 +53,45 @@ class DataLibrarySpec extends FreeSpec with WebBrowserSpec with UserFixtures wit
     }
   }
 
+  "For a dataset with facets" in withWebDriver { implicit driver =>
```

Comment:
does this test only test for the existence of Facets on Data Library page and nothing else?